appserver/extras/arquillian-containers/payara-embedded/src/test/resources/gfconfigs/instanceRoot/config/readme.txt

```diff
@@ -0,0 +1 @@
+This directory is meant to be the config directory of a Glassfish instanceRoot. It contains a blank domain.xml file for the purpose of testing.
```

Comment:
Hey, I went through all readme.txt files and changed it. How did this one slip through? Oh well.